File: 백준 Study/num1330.py
# 백준 1330번 두 정수 A와 B가 주어졌을 때, A와 B를 비교하는 프로그램을 작성하시오.

A, B = input().split()
if int(A) > int(B) :
    print(">")
elif int(A) < int(B) :
    print("<")
elif int(A) == int(B) :
    print("==")

File: 백준 Study/조건문/num1330.java
import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.io.IOException;
public class num1330 {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        String [] arr = br.readLine().split(" ");
        int A = Integer.parseInt(arr[0]);
        int B = Integer.parseInt(arr[1]);

        if(A > B) {
            System.out.println('>');    
        }
        else if(A < B) {
            System.out.println('<');    
        }
        else if(A == B) {
            System.out.println("=="); 
        }
    }
}
